nav2_bringup/launch/navigation_launch.py

```diff
@@ -61,11 +61,13 @@ def generate_launch_description():
     param_substitutions = {
         'autostart': autostart}
 
-    configured_params = RewrittenYaml(
-        source_file=params_file,
-        root_key=namespace,
-        param_rewrites=param_substitutions,
-        convert_types=True)
+    configured_params = ParameterFile(
```

Comment:
Totally approve of this action, but don't you need to do this for all instances of `RewrittenYaml`, not just this one so it could be used for anywhere this kind of pattern may be useful?

Reply:
Updated.